shared/src/persistence/dynamo/notifications/getDispatchNotification.js

```diff
@@ -0,0 +1,26 @@
+const { query } = require('../../dynamodbClientService');
+
+/**
+ * getDispatchNotification
+ *
+ * @param {object} providers
+ * @param {object} providers.applicationContext the application context
+ * @param {string} providers.topic the topic of the dispatch being sent
+ * @returns {array} an array of the results returned by the query
+ */
+exports.getDispatchNotification = async ({ applicationContext, topic }) =>
+  query({
+    ExpressionAttributeNames: {
+      '#pk': 'pk',
+      '#sk': 'sk',
+      '#ttl': 'ttl',
+    },
+    ExpressionAttributeValues: {
+      ':currentEpoch': Date.now() / 1000,
```

Comment:
I'm not totally familiar with the reason for the filter expression or this expression for currentEpoch, would it be possible to add a unit test to help document that? Just a thought

Reply:
Updated, let me know what you think.